Assets/Scripts/Characters/Player/SpeedUpAbility.cs

```diff
@@ -0,0 +1,51 @@
+﻿using UnityEngine;
+
+public class SpeedUpAbility : PlayerAbility
+{
+    public override void Activate()
+    {
+        isActive = true;
+        abilityDurationRemaining = abilityDuration;
+    }
+
+    public override void Deactivate()
+    {
+        
+    }
+
+    public override void ForceBreak()
+    {
+
+    }
+
+    public override bool IsActive
+    {
+        get { return isActive; }
+    }
+
+    public override int EnergyCost
+    {
+        get { return energyCost; }
+    }
+
+    private void Start() {
+        isActive = false;
+        energyCost = 50;
+    }
+
+    private void FixedUpdate() {
```

Comment:
Would be better if Ability Manager called Update function manually to reduce "magic" Monobehaviour Update() which is pretty slow